packages/nc-gui/components/smartsheet/header/VirtualCell.vue

```diff
@@ -153,11 +153,11 @@ const openDropDown = (e: Event) => {
   >
     <LazySmartsheetHeaderVirtualCellIcon v-if="column && !props.hideIcon" />
 
-    <a-tooltip placement="bottom">
+    <a-tooltip placement="bottom" :title="column.title">
```

Comment:
Use NcTooltip component

Reply:
**Fixed in below commit**

[fix : Used NcTooltip](https://github.com/nocodb/nocodb/pull/7114/commits/9207eda4c98368d9b747fb3ccff4a37b70b60fc7)
[9207eda](https://github.com/nocodb/nocodb/pull/7114/commits/9207eda4c98368d9b747fb3ccff4a37b70b60fc7)